internal/config/config.go

```diff
@@ -15,6 +15,7 @@ type ImageBuilderConfig struct {
 	ComposerClientId      string `env:"COMPOSER_CLIENT_ID"`
 	ComposerOfflineToken  string `env:"COMPOSER_OFFLINE_TOKEN"`
 	ComposerClientSecret  string `env:"COMPOSER_CLIENT_SECRET"`
+	ComposerTokener       string `env:"COMPOSER_TOKENER"`
```

Comment:
What is this for?

All the input configuration for tokener is already present. I am not following what is the idea here.